config.map

```diff
@@ -10,6 +10,7 @@ SCRAM_ARCH=slc7_amd64_gcc12;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_14_1_X/m
 SCRAM_ARCH=el9_amd64_gcc12;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_14_1_X/cs9;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_14_1_MULTIARCHSV4_X;DOCKER_IMG=cmssw/el9;BUILD_HOUR=24;MULTIARCH_OPTS=x86-64-v4;
 SCRAM_ARCH=el8_amd64_gcc12;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_14_1_X/master;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_14_1_SKYLAKEAVX512_X;DOCKER_IMG=cmssw/el8;BUILD_HOUR=23;BUILD_DAY=0,4;ADDITIONAL_TESTS=igprof-pp,profiling;MULTIARCH_OPTS=haswell,skylake-avx512;DISABLED=1;
 SCRAM_ARCH=el8_amd64_gcc12;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_14_1_X/master;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_14_1_GPU_X;DOCKER_IMG=cmssw/el8;BUILD_HOUR=23;
+SCRAM_ARCH=el8_amd64_gcc12;PKGTOOLS_TAG=V00-34-XX;CMSDIST_TAG=IB/CMSSW_14_1_X/master;RELEASE_BRANCH=master;RELEASE_QUEUE=CMSSW_14_1_ROCM_X;DOCKER_IMG=cmssw/el8;BUILD_HOUR=23;
```

Comment:
@aandvalenzuela , for now change the BUILD_HOUR to 24. so that this IB does not trigger automatically. We can manually build it so see how it goes. We might need to adjust few jenkins jobs to use correct node to run it e.g. https://cmssdt.cern.ch/jenkins/job/ib-validation need changes